CONTRIBUTING.rst

```diff
@@ -161,9 +161,15 @@ An example of that:
 Semantic release formulas
 -------------------------
 
+These formulas are already compatible with semantic-release *and* have global ``CODEOWNERS`` defined:
+
+#. `libvirt-formula <https://github.com/saltstack-formulas/libvirt-formula>`_
+#. `salt-formula <https://github.com/saltstack-formulas/salt-formula>`_
+#. `template-formula <https://github.com/saltstack-formulas/template-formula>`_
+
 These formulas are already compatible with semantic-release:
 
-#. `apache-formula <https://github.com/saltstack-formulas/apache-formula>`_
+4. `apache-formula <https://github.com/saltstack-formulas/apache-formula>`_
```

Comment:
a typo maybe?

Reply:
Good eyes, @daks!  Actually that's on purpose, it's to control the numbering for the second part of the list, so that it continues from the first.

Reply:
As you can see here: https://github.com/saltstack-formulas/.github/blob/3dd85fa1b9b75c634daff0b56f9572d2fbd55b25/CONTRIBUTING.rst#semantic-release-formulas.